### ISSUE TITLE
add how to deploy ziti in x cloud

### ISSUE THREAD

**Issue by philipleonardgriffiths** [openziti/ziti-doc#407]
Deployment of OpenZiti into various cloud. I put this as a low priority as we can focus initial work on K8S cloud implementations. 

Prio: Low.

**Comment by qrkourier** [comment]
Is this about self-hosting controllers and routers?

**Comment by philipleonardgriffiths** [comment]
Yes, it would be. How to get started on various clouds when self-hosting. The most common ones I have come across are AWS, Azure, GCP, OCI and Digital Ocean.
